### PR TITLE
add tesolin2023hint

### PR DESCRIPTION
**Pull request by pedropaulofb**

### CODE REVIEW

File: models/tesolin2023hint/meadata.yaml [hunk 1]
```diff
@@ -0,0 +1,31 @@
+title: Heterogeneous wIreless Network onTology
+acronym: hint
```
Comment (tgoprince) [inline models/tesolin2023hint/meadata.yaml:2]: Is this acronym really lower case?

File: models/tesolin2023hint/meadata.yaml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+ - https://dblp.org/pid/166/2068
+ - https://dblp.org/pid/342/7506
+ - https://dblp.org/pid/125/7740
+ - https://dblp.org/pid/c/MariaClaudiaReisCavalcanti
+keyword:
+ - mobile networks
+ - seamless handover
+ - telecommunications
+theme: Class T - Technology
+editorialNote: This model contains a n-ary relation. These relations are not supported by the current JSON exportation feature. Hence, to be able to generate the ontology.json file, this n-ary relation was temporarily excluded (but it is kept in the vpp file).
```
Comment (tgoprince) [inline models/tesolin2023hint/meadata.yaml:15]: Good call.

File: models/tesolin2023hint/meadata.yaml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+ - domain
+language: en
+designedForTask:
+ - conceptual clarification
+ - decision support system
+ - ontological analysis
+context:
+ - research
+source:
+ - https://doi.org/10.1016/j.future.2023.08.008
```
Comment (tgoprince) [inline models/tesolin2023hint/meadata.yaml:26]: Should not we link both publications in the bib file here?

**Review by tgoprince — CHANGES_REQUESTED** [review]
There are no TTL files. Please see other comments.

File: models/tesolin2023hint/meadata.yaml (hunk 1 above)
Comment (pedropaulofb) [reply models/tesolin2023hint/meadata.yaml:2]: That is how it was informed by the author. To be honest, I am not sure if intentionally or not. Do you think we should modify?

File: models/tesolin2023hint/meadata.yaml (hunk 3 above)
Comment (pedropaulofb) [reply models/tesolin2023hint/meadata.yaml]: Yes, it was my fault. The second identifier was added.

Comment (pedropaulofb) [reply models/tesolin2023hint/meadata.yaml]: ontology.ttl file added

**Review by tgoprince — APPROVED** [review]

**Pull request merged**